controller/server/third_party_service_storage.go

```diff
@@ -13,19 +14,21 @@ import (
 	"k8s.io/client-go/1.5/pkg/runtime"
 )
 
-type ThirdPartyServiceStorage struct {
+type thirdPartyServiceStorage struct {
 	watcher *watch.Watcher
 }
 
+// NewThirdPartyServiceStorage creates an instance of ServiceStorage
+// backed by Kubernetes third-party resources.
 func NewThirdPartyServiceStorage(w *watch.Watcher) ServiceStorage {
-	return &ThirdPartyServiceStorage{
+	return &thirdPartyServiceStorage{
 		watcher: w,
 	}
 }
 
-var _ ServiceStorage = (*ThirdPartyServiceStorage)(nil)
+var _ ServiceStorage = (*thirdPartyServiceStorage)(nil)
```

Comment:
if this is verifying that `*ThirdPartyServiceStorage` is a `ServiceStorage`, that should be left for test code. also, `NewThirdPartyServiceStorage` accomplishes as well